pkg/common/packageregistry.go

```diff
@@ -0,0 +1,31 @@
+package common
+
+import (
+	"fmt"
+
+	packagev1 "buf.build/gen/go/safedep/api/protocolbuffers/go/safedep/messages/package/v1"
+	"github.com/safedep/dry/packageregistry"
+)
+
+const errorScope = "Failed to resolve package latest version"
```

Comment:
```suggestion
const errorScope = "failed to resolve package latest version"
```

Go error convention is to use lower case